src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/InterpreterLanguage.java

```diff
@@ -0,0 +1,326 @@
+    RMM_RMM("rmm-rmm", "Roma", ""),
+    RUM_RUM("rum-rum", "Romanian", ""),
+    ROM_ROM("rom-rom", "Romany", ""),
+    CGG_CGG("cgg-cgg", "Rukiga", ""),
+    NYO_NYO("nyo-nyo", "Runyoro", ""),
+    RUS_RUS("rus-rus", "Russian", ""),
+    TTJ_TTJ("ttj-ttj", "Rutoro", ""),
+    SKR_SKR("skr-skr", "Saraiki (Seraiki)", ""),
+    KRN_KRN("krn-krn", "Sarpo", ""),
+    HBS_HBS("hbs-hbs", "Serbo-Croatian", ""),
+    TSN_TSN("tsn-tsn", "Setswana", ""),
+    SCL_SCL("scl-scl", "Shina", ""),
+    SNA_SNA("sna-sna", "Shona", ""),
+    SND_SND("snd-snd", "Sindhi", ""),
+    SIN_SIN("sin-sin", "Sinhalese", ""),
+    SLO_SLO("slo-slo", "Slovak", ""),
+    SLV_SLV("slv-slv", "Slovenian", ""),
+    SOM_SOM("som-som", "Somali", ""),
+    SPA_SPA("spa-spa", "Spanish", ""),
+    SUS_SUS("sus-sus", "Susu", ""),
+    SWA_SWA("swa-swa", "Swahili", ""),
+    SWA_KIBAJUNI("swa-kibajuni", "Swahili", "Kibajuni"),
+    SWA_BRAVANESE("swa-bravanese", "Swahili", "Bravanese"),
+    SWE_SWE("swe-swe", "Swedish", ""),
+    SYL_SYL("syl-syl", "Sylheti", ""),
+    TGL_TGL("tgl-tgl", "Tagalog", ""),
+    TAI_TAI("tai-tai", "Taiwanese", ""),
+    TAM_TAM("tam-tam", "Tamil", ""),
+    TEL_TEL("tel-tel", "Telugu", ""),
+    TEM_TEM("tem-tem", "Temne", ""),
+    THA_THA("tha-tha", "Thai", ""),
+    TIB_TIB("tib-tib", "Tibetan", ""),
+    TIG_TIG("tig-tig", "Tigre", ""),
+    TIR_TIR("tir-tir", "Tigrinya", ""),
+    DON_DON("don-don", "Toura", ""),
+    TUR_TUR("tur-tur", "Turkish", ""),
+    TUK_TUK("tuk-tuk", "Turkmen", ""),
+    TWI_TWI("twi-twi", "Twi", ""),
+    UIG_UIG("uig-uig", "Uighur", ""),
+    UKR_UKR("ukr-ukr", "Ukrainian", ""),
+    URD_URD("urd-urd", "Urdu", ""),
+    URH_URH("urh-urh", "Urohobo", ""),
+    UZB_UZB("uzb-uzb", "Uzbek", ""),
+    VIE_VIE("vie-vie", "Vietnamese", ""),
+    VSA_VSA("vsa-vsa", "Visayan", ""),
+    WEL_WEL("wel-wel", "Welsh", ""),
+    WOL_WOL("wol-wol", "Wolof", ""),
+    XHO_XHO("xho-xho", "Xhosa", ""),
+    YOR_YOR("yor-yor", "Yoruba", ""),
+    ZAG_ZAG("zag-zag", "Zaghawa", ""),
+    ZZA_ZZA("zza-zza", "Zaza", ""),
+    ZUL_ZUL("zul-zul", "Zulu", "");
+
+    private final String key;
+    private final String language;
+    private final String dialect;
+
+    public static InterpreterLanguage getInterpreterLanguageByLanguageAndDialect(String value, String dialect) {
+        for (InterpreterLanguage il : InterpreterLanguage.values()) {
+            if (il.getLanguage().equals(value) && il.getDialect().equals(dialect)) {
+                return il;
+            }
+        }
+        return null;
+    }
+
+    public static InterpreterLanguage getLanguageAndConvert(String languages){
+        return Arrays.stream(InterpreterLanguage.values())
+            .filter(c -> c.getLanguage().equals(languages))
+            .findFirst()
+            .orElse(conditionNotMet(languages));
+    }
+
+    public static InterpreterLanguage conditionNotMet(String languages){
+        switch (languages){
```

Comment:
Why not add this to the enum as a separate field instead of doing this?

Reply:
Easier to do this and doesn't make sense to add a new filed to the enum as then we may as well do it for every single one which would take days.